Documentation/security/network/proxy/envoy.rst

```diff
@@ -36,6 +36,7 @@ instead of running it from within the Cilium Agent Pod.
 The communication between Cilium agent and Envoy proxy takes place via UNIX domain sockets in both deployment modes.
 Be that streaming the access logs (e.g. L7 Protocol Visibility), updating the configuration via
 `xDS <https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol>`_ or accessing the admin interface.
+Due to the use of UNIX domain sockets, Envoy DaemonSet and the Cilium Agent need to have compatible types when SELinux is enabled on the host. This is the case if not specified otherwise, both using the highly privileged type: ``spc_t``. SELinux is enabled by default on Red Hat OpenShift Container Platform.
```

Comment:
Nice :+1: 

> This is the case if not specified otherwise

Did this change compared to older OpenShift releases?

Reply:
I just tested on 4.17. I have older dumps where the envoy daemonset is running on 4.16 and Cilium connectivity tests were successful

Reply:
ok. just wondering because this was an issue in the past (also for other areas of Cilium (e.g. Hubble) where pod-to-pod communication was done via unix domain sockets).

But glad if this works!